src/javascripts/Services/referencedByViewColumnsService.js

```diff
@@ -0,0 +1,114 @@
+ngapp.service('referencedByViewColumnsService', function(settingsService) {
+    let service = this,
+        settings = settingsService.settings;
+
+    let longNameColumn = {
+        label: 'Record',
+        canSort: true,
+        getData: function(node, xelib) {
+            if (node.element_type === xelib.etMainRecord) {
```

Comment:
I'm pretty sure the referencedByView is never going to display any nodes other than main record nodes, so this kind of conditional isn't really necessary, is it?